source/OctoPack.Tests.NonGitVersionAssembly/ClassInAssemblyWhereGitVersionIsNotUsed.cs

```diff
@@ -1,12 +0,0 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
-namespace OctoPack.Tests.NonGitVersionAssembly
-{
-    public class ClassInAssemblyWhereGitVersionIsNotUsed
```

Comment:
Renamed to [ClassInAssemblyWhereGitVersionIsNotUsed1](https://github.com/OctopusDeploy/OctoPack/pull/99/files#diff-a53d4110ad6b55bf952277fabf61e0bd)